{"version": 2, "width": 140, "height": 20, "timestamp": 1711006074, "idle_time_limit": 2.0, "env": {"SHELL": "/usr/bin/zsh", "TERM": null}, "title": "step-06-SH"}
[0.001863, "o", "sh: 1: source: not found\r\nstep-06-SH.sh: not found\r\n"]
[0.789757, "o", "[2024-03-21 07:27:54,791] ERROR [AdminClient clientId=adminclient-1] Connection to node -1 (broker-sni-gateway1main1.gateway-sni.conduktor.local/192.168.240.7:6969) failed authentication due to: SSL handshake failed (org.apache.kafka.clients.NetworkClient)\r\n"]
[0.792022, "o", "[2024-03-21 07:27:54,792] WARN [AdminClient clientId=adminclient-1] Metadata update failed due to authentication error (org.apache.kafka.clients.admin.internals.AdminMetadataManager)\r\norg.apache.kafka.common.errors.SslAuthenticationException: SSL handshake failed\r\nCaused by: javax.net.ssl.SSLHandshakeException: PKIX path validation failed: java.security.cert.CertPathValidatorException: signature check failed\r\n\tat java.base/sun.security.ssl.Alert.createSSLException(Alert.java:131)\r\n\tat java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:360)\r\n\tat java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:303)\r\n\tat java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:298)\r\n\tat java.base/sun.security.ssl.CertificateMessage$T13CertificateConsumer.checkServerCerts(CertificateMessage.java:1357)\r\n\tat java.base/sun.security.ssl.CertificateMessage$T13CertificateConsumer.onConsumeCertificate(CertificateMessage.java:1232)\r\n\tat java.base/sun.security.ssl.CertificateMessage$T13CertificateConsumer.consume(CertificateMessage.java:1175)\r\n\tat java.base/sun.security.ssl.SSLHandshake.consume(SSLHandshake.java:392)\r\n\tat java.base/sun.security.ssl.HandshakeContext.dispatch(HandshakeContext.java:443)\r\n\tat java.base/sun.security.ssl.SSLEngineImpl$DelegatedTask$DelegatedAction.run(SSLEngineImpl.java:1076)\r\n\tat java.base/sun.security.ssl.SSLEngineImpl$DelegatedTask$DelegatedAction.run(SSLEngineImpl.java:1063)\r\n\tat java.base/java.security.AccessController.doPrivileged(Native Method)\r\n\tat java.base/sun.security.ssl.SSLEngineImpl$DelegatedTask.run(SSLEngineImpl.java:1010)\r\n\tat org.apache.kafka.common.network.SslTransportLayer.runDelegatedTasks(SslTransportLayer.java:435)\r\n\tat org.apache.kafka.common.network.SslTransportLayer.handshakeUnwrap(SslTransportLayer.java:523)\r\n\tat org.apache.kafka.common.network.SslTransportLayer.doHandshake(SslTransportLayer.java:373)\r\n\tat org.apache.kafka.common.network.SslTransportLayer.handshake(SslTransportLayer.java:293)\r\n\tat org.apache.kafka.common.network.KafkaChannel.prepare(KafkaChannel.java:178)\r\n\tat org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:543)\r\n\tat org.apache.kafka.common.network.Selector.poll(Selector.java:481)\r\n\tat org.apache.kafka.clients.NetworkClient.poll(NetworkClient.java:571)\r\n\tat org.apache.kafka.clients.admin.KafkaAdminClient$AdminClientRunnable.processRequests(KafkaAdminClient.java:1381)\r\n\tat org.apache.kafka.clients.admin.KafkaAdminClient$AdminClientRunnable.run(KafkaAdminClient.java:1312)\r\n\tat java.base/java.lang.Thread.run(Thread.java:829)\r\nCaused by: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: signature check failed\r\n\tat java.base/sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:369)\r\n\tat java.base/sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:275)\r\n\tat java.base/sun.security.validator.Validator.validate(Validator.java:264)\r\n\tat java.base/sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:313)\r\n\tat java.base/sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:276)\r\n\tat java.base/sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:141)\r\n\tat java.base/sun.security.ssl.CertificateMessage$T13CertificateConsumer.checkServerCerts(CertificateMessage.java:1335)\r\n\t... 19 more\r\nCaused by: java.security.cert.CertPathValidatorException: signature check failed\r\n\tat java.base/sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:135)\r\n\tat java.base/sun.security.provider.certpath.PKIXCertPathValidator.validate(PKIXCertPathValidator.java:224)\r\n\tat java.base/sun.security.provider.certpath.PKIXCertPathValidator.validate(PKIXCertPathValidator.java:144)\r\n\tat java.base/sun.security.provider.certpath.PKIXCertPathValidator.engineValidate(PKIXCertPathValidator.java:83)\r\n\tat java.base/java.security.cert.CertPathValidator.validate(CertPathValidator.java:309)\r\n\tat java.base/sun.security.validator.PKI"]
[0.792079, "o", "XValidator.doValidate(PKIXValidator.java:364)\r\n\t... 25 more\r\nCaused by: java.security.SignatureException: Signature does not match.\r\n\tat java.base/sun.security.x509.X509CertImpl.verify(X509CertImpl.java:450)\r\n\tat java.base/sun.security.provider.certpath.BasicChecker.verifySignature(BasicChecker.java:166)\r\n\tat java.base/sun.security.provider.certpath.BasicChecker.check(BasicChecker.java:147)\r\n\tat java.base/sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:125)\r\n\t... 30 more\r\n"]
[0.793042, "o", "Error while executing topic command : SSL handshake failed\r\n"]
[0.793374, "o", "[2024-03-21 07:27:54,795] ERROR org.apache.kafka.common.errors.SslAuthenticationException: SSL handshake failed\r\nCaused by: javax.net.ssl.SSLHandshakeException: PKIX path validation failed: java.security.cert.CertPathValidatorException: signature check failed\r\n\tat java.base/sun.security.ssl.Alert.createSSLException(Alert.java:131)\r\n\tat java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:360)\r\n\tat java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:303)\r\n\tat java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:298)\r\n\tat java.base/sun.security.ssl.CertificateMessage$T13CertificateConsumer.checkServerCerts(CertificateMessage.java:1357)\r\n\tat java.base/sun.security.ssl.CertificateMessage$T13CertificateConsumer.onConsumeCertificate(CertificateMessage.java:1232)\r\n\tat java.base/sun.security.ssl.CertificateMessage$T13CertificateConsumer.consume(CertificateMessage.java:1175)\r\n\tat java.base/sun.security.ssl.SSLHandshake.consume(SSLHandshake.java:392)\r\n\tat java.base/sun.security.ssl.HandshakeContext.dispatch(HandshakeContext.java:443)\r\n\tat java.base/sun.security.ssl.SSLEngineImpl$DelegatedTask$DelegatedAction.run(SSLEngineImpl.java:1076)\r\n\tat java.base/sun.security.ssl.SSLEngineImpl$DelegatedTask$DelegatedAction.run(SSLEngineImpl.java:1063)\r\n\tat java.base/java.security.AccessController.doPrivileged(Native Method)\r\n\tat java.base/sun.security.ssl.SSLEngineImpl$DelegatedTask.run(SSLEngineImpl.java:1010)\r\n\tat org.apache.kafka.common.network.SslTransportLayer.runDelegatedTasks(SslTransportLayer.java:435)\r\n\tat org.apache.kafka.common.network.SslTransportLayer.handshakeUnwrap(SslTransportLayer.java:523)\r\n\tat org.apache.kafka.common.network.SslTransportLayer.doHandshake(SslTransportLayer.java:373)\r\n\tat org.apache.kafka.common.network.SslTransportLayer.handshake(SslTransportLayer.java:293)\r\n\tat org.apache.kafka.common.network.KafkaChannel.prepare(KafkaChannel.java:178)\r\n\tat org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:543)\r\n\tat org.apache.kafka.common.network.Selector.poll(Selector.java:481)\r\n\tat org.apache.kafka.clients.NetworkClient.poll(NetworkClient.java:571)\r\n\tat org.apache.kafka.clients.admin.KafkaAdminClient$AdminClientRunnable.processRequests(KafkaAdminClient.java:1381)\r\n\tat org.apache.kafka.clients.admin.KafkaAdminClient$AdminClientRunnable.run(KafkaAdminClient.java:1312)\r\n\tat java.base/java.lang.Thread.run(Thread.java:829)\r\nCaused by: sun.security.validator.ValidatorException: PKIX path validation failed: java.security.cert.CertPathValidatorException: signature check failed\r\n\tat java.base/sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:369)\r\n\tat java.base/sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:275)\r\n\tat java.base/sun.security.validator.Validator.validate(Validator.java:264)\r\n\tat java.base/sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManagerImpl.java:313)\r\n\tat java.base/sun.security.ssl.X509TrustManagerImpl.checkTrusted(X509TrustManagerImpl.java:276)\r\n\tat java.base/sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(X509TrustManagerImpl.java:141)\r\n\tat java.base/sun.security.ssl.CertificateMessage$T13CertificateConsumer.checkServerCerts(CertificateMessage.java:1335)\r\n\t... 19 more\r\nCaused by: java.security.cert.CertPathValidatorException: signature check failed\r\n\tat java.base/sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:135)\r\n\tat java.base/sun.security.provider.certpath.PKIXCertPathValidator.validate(PKIXCertPathValidator.java:224)\r\n\tat java.base/sun.security.provider.certpath.PKIXCertPathValidator.validate(PKIXCertPathValidator.java:144)\r\n\tat java.base/sun.security.provider.certpath.PKIXCertPathValidator.engineValidate(PKIXCertPathValidator.java:83)\r\n\tat java.base/java.security.cert.CertPathValidator.validate(CertPathValidator.java:309)\r\n\tat java.base/sun.security.validator.PKIXValidator.doValidate(PKIXValidator.java:364)\r\n\t... 25 more\r\nCaused by: java.security.SignatureException: Signature does not match.\r\n\tat java.base/sun.s"]
[0.793415, "o", "ecurity.x509.X509CertImpl.verify(X509CertImpl.java:450)\r\n\tat java.base/sun.security.provider.certpath.BasicChecker.verifySignature(BasicChecker.java:166)\r\n\tat java.base/sun.security.provider.certpath.BasicChecker.check(BasicChecker.java:147)\r\n\tat java.base/sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:125)\r\n\t... 30 more\r\n (kafka.admin.TopicCommand$)\r\n"]
